test-plan/eventing/event-delivery-conformance.md

```diff
@@ -0,0 +1,979 @@
+# Event Delivery
+
+From: https://github.com/knative/specs/blob/main/specs/eventing/control-plane.md#event-delivery
+
+
+From the Spec:
+
+>> Once a Trigger or Subscription has decided to deliver an event, it MUST do the
+following:
+>>
+>> 1. Read the resolved URLs and delivery options from the object's `status`
+   fields.
+>>
+>> 1. Attempt delivery to the `status.subscriberUri` URL following the
+   [data plane contract](./data-plane.md).
+>>
+>>    1. If the event delivery fails with a retryable error, it MUST be retried up
+      to `retry` times (subject to congestion control), following the
+      `backoffPolicy` and `backoffDelay` parameters if specified.
+>>
+>> 1. If the delivery attempt is successful (either the original request or a
+   retry) and no event is returned, the event delivery is complete.
+>>
+>> 1. If the delivery attempt is successful (either the original request or a
+   retry) and an event is returned in the reply, the reply event MUST be
+   delivered to the `status.replyUri` destination (for Subscriptions) or added
+   to the Broker for processing (for Triggers). If `status.replyUri` is not
+   present in the Subscription, the reply event MUST be dropped.
+>>
+>>    i. For Subscriptions, if delivery of the reply event fails with a retryable
+      error, the entire delivery of the event to MUST be retried up to `retry`
+      times (subject to congestion control), following the `backoffPolicy` and
+      `backoffDelay` parameters if specified.
+>>
+>> 5. If an event (either the initial event or a reply) cannot be delivered, the
+   event MUST be delivered to the `deadLetterSink` in the delivery options. If
+   no `deadLetterSink` is specified, the event is dropped.
+>>
+>>    The implementation MAY set additional attributes on the event or wrap the
+   failed event in a "failed delivery" event; this behavior is not (currently)
+   standardized.
+>>
+>>    If delivery of the dead-letter event fails with a retryable error, the
+   delivery to the `deadLetterSink` SHOULD be retried up to `retry` times,
+   following the `backoffPolicy` and `backoffDelay` parameters if specified.
+   Alternatively, implementations MAY use an equivalent internal mechanism for
+   delivery (for example, if the `ref` form of `deadLetterSink` points to a
+   compatible implementation).
+
+# Testing Event Delivery Conformance
+
+This set of tests provide conformance on the event delivery capabilities of Subscription/Channels and Trigger/Brokers resources. All referenced commands should be executed from the same directory this documentation is to be found.
+
```

Comment:
Can you add a paragraph stating where to find the yaml files, to be consistent with https://github.com/knative/specs/blob/main/test-plan/eventing/broker-lifecycle-conformance.md#testing-broker-lifecycle-conformance